banderwagon/testdata/fuzz/FuzzDeserializationCompressed/caf81e9797b19c76

```diff
@@ -0,0 +1,2 @@
+go test fuzz v1
```

Comment:
I commit these fuzzer corpus since now `go test` will automatically run these cases that caused panics or inconsistencies before.